app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_arr = command.split()
+    action = command_arr[0]
+    source = command_arr[1]
+    destination = command_arr[2]
+
+    if source == destination and action == "mv":
+        return
+
+    if "/" not in destination and action == "mv":
+        os.rename(source, destination)
+        return
+
+    if destination.endswith("/"):
```

Comment:
Why do we need this?

Reply:
If destination ends with "/" - we just have a path without name of new file.
So, name of new file will be name of old file.
Else, we get name of new file from the end of "destination".